pg_analytics/src/hooks/query.rs

```diff
@@ -20,7 +24,7 @@ pub trait Query {
     fn get_ast(self, query_string: &str) -> Result<VecDeque<parser::Statement>, ParadeError>;
 
     // Parses the query string into a DataFusion LogicalPlan
-    fn get_logical_plan(self, query_string: &str) -> Result<LogicalPlan, ParadeError>;
+    fn get_logical_plan(self, query_string: &str) -> Result<(LogicalPlan, bool), ParadeError>;
```

Comment:
It shouldn't be two different functions because you shouldn't be able to get a logical plan that hasn't had the UDFs properly replaced. I can see the tuple result being confusing, though - I'll wrap it in a struct.